package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "2.6.6",
+  "version": "2.6.7",
```

This is new functionality, so this should be a minor version bump
```suggestion
  "version": "2.7.0",
```